### PR TITLE
chore: add boilerplate

### PR DESCRIPTION
**Pull request by open-dynaMIX**
This commit ...

 - ...adds some boilerplate
 - ...replaces travis with gh actions
 - ...blackifies the code
 - ...adds django-extensions

A second commit updates all dependencies.

### CODE REVIEW

File: emeis/emeis_metadata.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"""Used to single sourcing metadata about caluma."""
```
Comment (winged) [inline emeis/emeis_metadata.py:1]: s/caluma/emeis/

Comment (open-dynaMIX) [reply emeis/emeis_metadata.py]: :) Done.

**Comment by open-dynaMIX** [comment]
> What I'd like to see is the content of the proposal from [projectcaluma/caluma#1047](https://github.com/projectcaluma/caluma/issues/1047) copied over to the docs of this project. This way we already have a good starting point for more documentation

Done.

**Comment by open-dynaMIX** [comment]
@winged I've also included the LICENSE now.

**Comment by winged** [comment]
> @winged I've also included the LICENSE now.

Hah, good catch!

**Pull request merged**